packages/odyssey-storybook/src/contributing/Documentation.stories.mdx

```diff
@@ -0,0 +1,207 @@
+    Behavior describes default behaviors included with components.
+  </dd>
+  <dd>
+    This section is not required if the component has limited or self-evident
+    behavior.
+  </dd>
+  <dt>Variants (Optional)</dt>
+  <dd>
+    This section decribes the differing configurations available for the
+    component.
+  </dd>
+  <dd>
+    These variants could be global or semantic changes (e.g. "Success" vs
+    "Danger") or different component assemblies (e.g. "With actions")
+  </dd>
+  <dt>States</dt>
+  <dd>This section covers all the states a component can find itself in.</dd>
+  <dd>
+    Pseudo-classed states requiring user interaction, specifically `:focus` and
+    `:hover` are not required if those states can be observed via quick user
+    interaction.
+  </dd>
+  <dt>Usage</dt>
+  <dd>
+    This is where we lay out the function of components and how they can be used
+    within a design/view/page.
+  </dd>
+  <dd>DOs and DONTs are appropriate for this section.</dd>
+  <dt>Content Guidelines</dt>
+  <dd>
+    This is where we lay out any requirements around acceptable content for use
+    within the component.
+  </dd>
+  <dd>
+    This includes both what a component may contain as well as how that content
+    should be written.
+  </dd>
+  <dd>DOs and DONTs are appropriate for this section.</dd>
+  <dt>Accessibility</dt>
+  <dd>
+    This section includes any guidelines required for compliance as well as
+    design rationale already leveraged to ensure compliance - both "please do
+    this" and "we do this because".
+  </dd>
+  <dt>Responsive Web Design (optional)</dt>
+  <dd>Any considerations for use or documentation.</dd>
+  <dd>
+    This section is rare today, but should include examples and rationale
+    similar to the Accessibility section.
+  </dd>
+  <dt>Globalization (optional)</dt>
+  <dd>
+    This should include any notes about use or cultural considerations, as well
+    as whether the component is lacking globalization support in some regard.
+  </dd>
+  <dt>References (optional)</dt>
+  <dd>
+    This section has two optional children: Related Components and Further
+    Reading
+  </dd>
+  <dt>Related Components (optional)</dt>
+  <dd>
+    Points the user toward any related components, possibly alternate solutions.
+  </dd>
+  <dd>
+    We do this in a few places today and it may be beneficial to include in more
+    places.
+  </dd>
+  <dt>Further Reading (optional)</dt>
+  <dd>
+    Again, included in some places, always optional, but could provide
+    additional rationale or education for our users.
+  </dd>
+  <dd>
+    These have been especially helpful when deviating from common dark patterns
+    or legacy solutions.
+  </dd>
+</dl>
+
+## Docs template
+
+```md
+# Landing Gear
+
+This component provides traction and cushion during take-off and landing.
+
+[Figma](http://figma.com/path/to/my-component) - [Source](http://github.com/okta/odyssey/path/to/source)
+
+## Anatomy
+
+**ANATOMY PLACEHOLDER**
```

Comment:
Im assuming this will get updated when we either include anatomy or decide to scrap it altogether, yeah?

Reply:
Once we have a solution in place, this will get updated to `<Anatomy src="x"/>` or a simple `<img src=""/>`.